src/graphql/helpers.ts

```diff
@@ -154,7 +154,7 @@ export function buildWhereQuery(fields, alias, where) {
 export async function fetchSpaces(args) {
   const { first = 20, skip = 0, where = {} } = args;
 
-  const fields = { id: 'string' };
+  const fields = { id: 'string', created_at: 'number' };
```

Comment:
we will pass exact timestamp to get a space?? why do we need this??

Also `created_at` is not added in `src/graphql/schema.gql` for inputs type of `Space` query, so it will throw error if we pass `created_at`

Reply:
column removed, will find another way from service to filter by date